library/replication/configure.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+. $(dirname $0)/../store/store.sh
+
+# WORKING via Swagger on NBFS 3.1:
+#
+# curl -X 'POST' \
+#   'https://lagoscl02mgmt.engba.veritas.com:14161/api/appliance/v1.0/disaster-recovery' \
+#   -H 'accept: application/json' \
+#   -H "Authorization: Bearer $token"
+#   -H 'Content-Type: application/json' \
+#   -d '{
+#   "targetManagementFqdn": "capetowncl02mgmt.engba.veritas.com",
```

Comment:
Are you sure we're supposed to be publishing the names of internal engineering servers? We go to great lengths to avoid including them in the products we ship.